pkgs/tools/networking/n2n/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, stdenv, fetchFromGitHub, autoconf, automake, pkg-config }:
+
+stdenv.mkDerivation rec {
+  pname = "n2n";
+  version = "2.8";
+
+  src = fetchFromGitHub {
+    owner = "ntop";
+    repo = "n2n";
+    rev = version;
+    hash = "sha256-2xJ8gYVZJZoKs6PZ/9GacgxQ+/3tmnRntT1AbPe1At4=";
+  };
+
+  nativeBuildInputs = [ autoconf automake pkg-config ];
+
+  preConfigure = ''
+    patchShebangs autogen.sh
+    ./autogen.sh
+  '';
+
+  # According to the README, some parts of the code significantly benefit from
+  # compiler optimizations and platform features such as NEON, SSE and AVX.
+  preferLocalBuild = true;
```

Comment:
```suggestion
```

This is meant to be used for ```This is appropriate for trivial builders where the cost of doing a download or remote build would exceed the cost of building locally.``` according to the nix docs. There are config flags for AVX if you really care about this feature.